projects/hslayers/src/components/sidebar/sidebar.service.ts

```diff
@@ -283,8 +284,11 @@ export class HsSidebarService {
   }
 
   setButtonVisibility() {
+    this.numberOfUnimportant = this.buttons.filter((button) => {
+      return button.important == false;
+    }).length;
     for (const button of this.buttons) {
-      const fits = this.fitsSidebar(button.panel);
+      const fits = button.important == false ? true : this.fitsSidebar(button);
```

Comment:
This seems realy counterintuitive, could you add a comment how it works?

Reply:
Agreed, I fixed that in last commit. Most likely I wanted to squash the last two ...looking at it right now
https://github.com/hslayers/hslayers-ng/pull/1573/commits/00867a21d780b685d6f8d95fb8b8724f422872e2#diff-d8dc65320ed0d747257320318008dc76f9053cbefd212192106eba5e05e8e63cR296

Reply:
Indeed, thanks for pointing that out. 
https://github.com/hslayers/hslayers-ng/pull/1573/commits/558151e103eba3bdcfdbe29623e9533a4f3cc67c#diff-d8dc65320ed0d747257320318008dc76f9053cbefd212192106eba5e05e8e63cR296